src/components/ChartTodo/ChartTodo.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Paper } from '@material-ui/core';
+import {BarChart,CartesianGrid ,XAxis,YAxis,Bar , ResponsiveContainer  , Tooltip, Text} from 'recharts'
+import classes from './ChartTodo.module.css';
+import axios from '../../axios-ListData';
+import withErrorHandler from '../../hoc/withErrorHandler/withErrorHandler';
+
+
+class ChartTodo extends Component {
+
+
+    state= {
+        todoList : [],
+
+    }
+
+    componentDidMount () {
+        axios.get('/ListTodo.json')
+             .then(response => {
+                let updateTodoData = [] ;
+                let todoData = Object.keys(response.data).map(item => {
+                    return {...response.data[item]  ,id : item}  })
+                    for( let key in todoData) {
+                        updateTodoData.push(todoData[key])
+                    }
```

Comment:
What is the difference between todoData and updateTodoData? they are the same right?